src/app/containers/MediaPlayer/helpers/metadata/index.test.js

```diff
@@ -1,91 +1,105 @@
-import mediaPlayerMetadata from '.';
+import {
+  mediaPlayerMetadata,
+  getThumbnailUri,
+  getMetadataBlock,
+  getMetadata,
+  getUploadDate,
+} from '.';
 import {
   validAresMediaVideoBlock,
   validAresMediaAudioBlock,
-  missingAresMediaMetadataBlock,
-  multipleAresMetadataBlock,
+  validAresMetadataBlock,
 } from '../../fixtureData';
+import { getType } from '#app/containers/ChartbeatAnalytics/utils';
 
-describe('mediaPlayerMetadata', () => {
-  it('returns correct video metadata', () => {
-    const metadata = mediaPlayerMetadata(validAresMediaVideoBlock);
-    const output = {
-      video: {
-        '@list': [
-          {
-            '@type': 'VideoObject',
-            name: 'Five things ants can teach us about management',
-            description:
-              'They may be tiny, but us humans could learn a thing or two from ants.',
-            duration: 191,
-            thumbnailUrl:
-              'https://ichef.test.bbci.co.uk/images/ic/1024x576/p01k6mtv.jpg',
-            uploadDate: 1540218932000,
-          },
-        ],
-      },
-    };
-    expect(metadata).toEqual(output);
+const embedSource =
+  'https://www.test.bbc.com/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6mtv/en-GB';
+
+describe('helper', () => {
+  describe('mediaPlayerMetadata', () => {
+    it('returns correct video metadata', () => {
+      const aresMediaBlocks = validAresMediaVideoBlock.model.blocks;
+      const metadata = mediaPlayerMetadata(aresMediaBlocks);
+      const output = {
+        '@context': 'http://schema.org',
+        '@type': 'VideoObject',
+        description:
+          'They may be tiny, but us humans could learn a thing or two from ants.',
+        duration: 'PT3M11S',
+        embedURL: null,
+        name: 'Five things ants can teach us about management',
+        thumbnailUrl:
+          'https://ichef.test.bbci.co.uk/images/ic/1024x576/p01k6mtv.jpg',
+        uploadDate: '2018-10-22T14:35:32.000Z',
+      };
+      expect(metadata).toEqual(output);
+    });
+
+    it('returns correct audio metadata', () => {
+      const aresMediaBlocks = validAresMediaAudioBlock.model.blocks;
+      const metadata = mediaPlayerMetadata(aresMediaBlocks);
```

Comment:
```suggestion
      const metadata = mediaPlayerMetadata(aresMediaBlocks, embedSource);
```
this would make `embedURL` on line46 not to be null

Reply:
Addressed and resolved